packages/grafana-toolkit/docker/grafana-plugin-ci/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM circleci/node:12-browsers
+USER root 
+WORKDIR /tmp
+
+# Install Go
+ADD https://dl.google.com/go/go1.14.linux-amd64.tar.gz /tmp
+RUN echo 08df79b46b0adf498ea9f320a0f23d6ec59e9003660b4c9c1ce8e5e2c6f823ca go1.14.linux-amd64.tar.gz | sha256sum --check --status
+RUN sudo tar -C /usr/local -xf go1.14.linux-amd64.tar.gz
```

Comment:
I don't think we need `sudo` in a Dockerfile (which is essentially a build script)?

Reply:
Yeah, probably not when running as root. You are right. I can remove it. The development started out as user circleci, which required that, and later moved over.